checker/manual/creating-a-checker.tex

```diff
@@ -516,23 +499,19 @@ \subsection{Declaratively specifying implicit annotations\label{declarative-type
 %END LATEX
 \begin{Verbatim}
   @SubtypeOf( { Nullable.class } )
-  @ImplicitFor(
-    types={TypeKind.PACKAGE},
-    typeClasses={AnnotatedPrimitiveType.class},
-    trees={
-      Tree.Kind.NEW_CLASS,
-      Tree.Kind.NEW_ARRAY,
-      Tree.Kind.PLUS,
+  @ImplicitFor(types = { TypeKind.PACKAGE, TypeKind.INT,
+                         TypeKind.BOOLEAN, TypeKind.CHAR,
+                         TypeKind.DOUBLE, TypeKind.FLOAT,TypeKind.LONG },
```

Comment:
Missing space after comma.

As a more minor comment, I would put PACKAGE on its own line and then have lines just with primitives.